zomes/holo_acorn/code/src/lib.rs

```diff
@@ -432,7 +436,48 @@ mod holo_acorn {
     fn fetch_agent_address() -> ZomeApiResult<Address> {
       Ok(AGENT_ADDRESS.clone())
     }
+    #[zome_fn("hc_public")]
+    fn history_of_goal()->ZomeApiResult<Vec<GetHistoryResponse>>{
```

Comment:
Let's make this accept a Goal address, and then just get the history for that single Goal
```suggestion
    fn history_of_goal(address: Address)->ZomeApiResult<Vec<GetHistoryResponse>>{
```